test/letters/containers/Main.unit.spec.jsx

```diff
@@ -1,55 +1,134 @@
 import React from 'react';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
-import _ from 'lodash';
+import sinon from 'sinon';
 
 import { Main } from '../../../src/js/letters/containers/Main';
 
+const getBenefitSummaryOptionsSpy = sinon.spy();
+const getLetterListSpy = sinon.spy();
+const getMailingAddressSpy = sinon.spy();
+const getAddressCountriesSpy = sinon.spy();
+const getAddressStateSpy = sinon.spy();
```

Comment:
You don't actually use these spies anywhere, right? Took me a while to notice that. I'd say just define `defaultProps.getMailingAddress()` and friends as empty arrow functions `() => {}` and then override them with spies in the tests where you check to see if they're called.

Reply:
@cvalarida Yeah, I was looking at a couple of different approaches and forgot to clean up :/
I'll move things around and push up a new commit